plugins/modules/net_tools/nmcli.py

```diff
@@ -1907,6 +1920,7 @@ def main():
             wifi=dict(type='dict'),
             wifi_sec=dict(type='dict', no_log=True),
             gsm=dict(type='dict'),
+            wireguard=dict(type='dict'),
```

Comment:
You also need to document this new option (see `DOCUMENTATION` at the top of the file).